index.rst

```diff
@@ -1,40 +1,378 @@
+Modern HTML, CSS and JavaScript allow for better, more innovative, and frictionless reading experiences than PDFs and Word documents can provide.
+Modern readers expect to be able to search for content, click a link, and read it immediately in the browser.
+
+Technotes are written in reStructuredText
+-----------------------------------------
+
+A fundamental requirement of the technote platform is that documents are written in a plain text format, and are then compiled into a website. This way, content is written in common text editors and hosted on GitHub for collaboration (the same workflow we already use for source code).
+Many plain text formats have been invented for writing, however, not all formats are ideal for technotes.
+For example, LaTeX, though common in astronomy, is tied too closely to PDF output and makes little sense for a web-native syntax.
+`Markdown <http://daringfireball.net/projects/markdown/>`_ is a hugely popular format among developers, in part because of its intuitive simplicity.
+However, this simplicity has prompted developers to `add non-standard extensions to the format <http://commonmark.org>`_ to their markup to add semantics and style to content.
+
+Instead, reStructuredText is an ideal markup format for technotes.
+reStructuredText's syntax and build process was designed to be extended, and in fact, those extensions are made in DM's primary language, Python.
+Because the Python community has adopted reStructuredText as its official markup language, a our technote platform can take advantage of excellent open source tools, such as the Sphinx build system and the Read the Docs documentation hosting site.
+For these same reasons, reStructuredText and the Sphinx build toolchain are also used by DM's new software documentation platform.
+This allows both documentation platforms to benefit from the same bespoke infrastructure development, such as SQuaRE's `documenteer`_ package.
+
+Technotes are single-page documents
+-----------------------------------
+
+This limitation is made purposefully to limit the scope of technotes, make it possible to easily print a technote in its entirely, and allow readers to use their browser's search feature to navigate a document.
+We are designing technote presentation to accommodate on-page navigation within a long document.
+
+Technotes are citeable
+----------------------
+
+Since they are standalone, self-encapsulated documents, technotes can be easily archived and assigned digital object identifiers, DOIs.
+DOIs allow technotes to be easily cited in scientific literature.
+The benefit of a citeable-trail of DM documentation is that LSST publications can more accurately describe DM's engineering work.
+We use Zenodo_ as an archive and DOI provider, taking advantage of its GitHub integration.
+
+Technotes are versioned
+-----------------------
+
+Unlike scientific publications that are published once, technotes can and should be updated when appropriate.
```

Comment:
The "can and should be updated when appropriate" phrasing makes it hard to say whether you approve of tech notes that provide information and then never change. Also may want to mention in the DOI tech note how DOIs deal with versioned documents.